murfi-rt-PyProject/scripts/feedback.sh

```diff
@@ -126,29 +79,39 @@ then
+    #cp $subj_dir/img/img-00002-00002.nii  $subj_dir/rest/$subj'_'$ses'_task-rest_'$run'_bold'.nii
+    #yes n | gzip $subj_dir/rest/$subj'_'$ses'_task-rest_'$run'_bold'.nii
     
-    #../scripts/img/img-00001-00211.nii ../scripts/img/img-00001-00212.nii ../scripts/img/img-00001-00213.nii ../scripts/img/img-00001-00214.nii ../scripts/img/img-00001-00215.nii ../scripts/img/img-00001-00216.nii ../scripts/img/img-00001-00217.nii ../scripts/img/img-00001-00218.nii ../scripts/img/img-00001-00219.nii ../scripts/img/img-00001-00220.nii ../scripts/img/img-00001-00221.nii ../scripts/img/img-00001-00222.nii ../scripts/img/img-00001-00223.nii ../scripts/img/img-00001-00224.nii ../scripts/img/img-00001-00225.nii ../scripts/img/img-00001-00226.nii ../scripts/img/img-00001-00227.nii ../scripts/img/img-00001-00228.nii ../scripts/img/img-00001-00229.nii ../scripts/img/img-00001-00230.nii ../scripts/img/img-00001-00231.nii ../scripts/img/img-00001-00232.nii ../scripts/img/img-00001-00233.nii ../scripts/img/img-00001-00234.nii ../scripts/img/img-00001-00235.nii ../scripts/img/img-00001-00236.nii ../scripts/img/img-00001-00237.nii ../scripts/img/img-00001-00238.nii ../scripts/img/img-00001-00239.nii ../scripts/img/img-00001-00240.nii ../scripts/img/img-00001-00241.nii ../scripts/img/img-00001-00242.nii ../scripts/img/img-00001-00243.nii ../scripts/img/img-00001-00244.nii ../scripts/img/img-00001-00245.nii ../scripts/img/img-00001-00246.nii ../scripts/img/img-00001-00247.nii ../scripts/img/img-00001-00248.nii ../scripts/img/img-00001-00249.nii 1.2
+    # get all volumes of resting data (no matter how many) merged into 1 .nii.gz file
+    # NOTE: the -00002 extensipn will likely need to be adjusted depending on where this scan falls in the protocol
+    fslmerge -tr $subj_dir/rest/$subj'_'$ses'_task-rest_'$run'_bold'.nii.gz $subj_dir/img/img-00002* 1.2
```

Comment:
@cccbauer -- might be helpful to discuss best practices here, i.e. which image pattern to default to pulling from the `$subj_dir/img` folder, or whether we may want to set arguments for this function

Reply:
We have to pull from the `img/` folder since here is where murfi puts them after acquisition, unless we copy them to another place

Reply:
@cccbauer thanks! Within the `img/` folder, do we want to hardcode this so we always pull `img-00001*` files, for example? Or should this be more flexible so we can choose whether to pull, for example `img-00001*`  or `img-00002*`, depending on the order of the runs (or if a resting-state scan has to be re-run for some reason)?

Reply:
ones the protocol is all set, this should not change, but lets make it flexible or, we could first count the `img-00001` files, and if the count is 250 then go ahead and `fslmerge` else, ask for the correct `img-` number

Reply:
I've set it to first look at the `img-00000` files for now, and if there are 2 or fewer, use `img-00001` files instead...definitely a placeholder we can update once we finalize the protocol